src/bundle/DependencyInjection/Configuration.php

```diff
@@ -1,21 +1,21 @@
 <?php
 
 /**
- * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  */
 declare(strict_types=1);
 
-namespace EzSystems\EzPlatformAutomatedTranslationBundle\DependencyInjection;
+namespace Ibexa\Bundle\AutomatedTranslation\DependencyInjection;
 
-use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware;
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\SiteAccessAware;
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 
 class Configuration extends SiteAccessAware\Configuration
 {
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder('ez_platform_automated_translation');
+        $treeBuilder = new TreeBuilder('ibexa_automated_translation');
         $rootNode = $treeBuilder->getRootNode();
         $systemNode = $this->generateScopeBaseNode($rootNode);
```

Comment:
TBH I would prefer Parsers for `ibexa.system.<scope>.<settings>` to this old extension point, but probably out of scope for rebranding.